joint_state_broadcaster/src/joint_state_broadcaster.cpp

```diff
@@ -0,0 +1,237 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
```suggestion
// Copyright 2021 ros2_control development team
```

I don't think OSRF contributed much for this. At least after the most recent changes where the framework is completely restructured.

@bmagyar what do you think?